### ISSUE TITLE
Can't Set Filtering Select Value

### ISSUE THREAD

**Issue by johhnywishwash** [Alfresco/Aikau#1057]
Noticed that I can't set the value on a **alfresco/forms/controls/FilteringSelect** inside the config element of the widget.  However it is possible to set it using the value object in the **alfresco/forms/Form**.

Example tested against 1.0.67

```
model.jsonModel = {
	widgets : [ {
		name : "alfresco/forms/Form",
		config : {
			okButtonPublishTopic : "SAVE_FORM",
			okButtonLabel : "Save",
			value : {
				FILTERING_SELECT_2 : "alex_sim"
			},
			widgets : [ {
				id : "FILTERING_SELECT_1",
				name : "alfresco/forms/controls/FilteringSelect",
				config : {
					fieldId : "FILTERING_SELECT_1",
					name : "person1",
					label : "Select a person",
					value : "alex_sim", //USERNAME VALUE
					optionsConfig : {
						queryAttribute : "label",
						labelAttribute : "label",
						valueAttribute : "value",
						publishTopic : "ALF_GET_FORM_CONTROL_OPTIONS",
						publishPayload : {
							resultsProperty : "options",
							url : url.context + "/proxy/alfresco/api/people",
							itemsAttribute : "people",
							labelAttribute : "userName",
							valueAttribute : "userName"
						}
					}
				}
			}, {
				id : "FILTERING_SELECT_2",
				name : "alfresco/forms/controls/FilteringSelect",
				config : {
					fieldId : "FILTERING_SELECT_2",
					name : "person2",
					label : "Select a person",
					optionsConfig : {
						queryAttribute : "label",
						labelAttribute : "label",
						valueAttribute : "value",
						publishTopic : "ALF_GET_FORM_CONTROL_OPTIONS",
						publishPayload : {
							resultsProperty : "options",
							url : url.context + "/proxy/alfresco/api/people",
							itemsAttribute : "people",
							labelAttribute : "userName",
							valueAttribute : "userName"
						}
					}
				}
			} ]
		}
	}, {
		name : "alfresco/logging/DebugLog"
	} ],
	services : [ "alfresco/services/OptionsService" ]
};
```

**Comment by aviriel** [comment]
Maybe it's the same issue, but FilteringSelect fields are not filled from hash when using "useHash: true" on the form. TextBox and Select are filled, but FilteringSelect ignores the value.

**Comment by draperd** [comment]
Apologies for the delay in responding to this issue - the Aikau team were on vacation, we will start looking through the backlog of issues from today - thanks for your patience!

**Comment by draperd** [comment]
I've raised https://issues.alfresco.com/jira/browse/AKU-972 to address this issue, as a bug it will be prioritized for the next sprint.

**Comment by draperd** [comment]
The fix for this is in the 1.0.69 release which is now available.
